src/Tribe/Promoter/Connector.php

```diff
@@ -54,7 +54,7 @@ public function authorize_with_connector( $user_id, $secret_key, $promoter_key,
 			'userId'       => $user_id,
 		];
 
-		$token = \Firebase\JWT\JWT::encode( $payload, $promoter_key, 'HS256' );
+		$token = \TEC\Common\Firebase\JWT\JWT::encode( $payload, $promoter_key, 'HS256' );
```

Comment:
Looks good, a suggestion to do use TEC\Common\Firebase\JWT\JWT as TEC_JWT and then use that here and below.